packages/design/tailwind/css/components.css

```diff
@@ -8,24 +8,51 @@
   focus:gi-outline-offset-0;
 }
 
+.gi-focus-visible-state-outline {
+  @apply focus-visible:gi-outline
+  focus-visible:gi-outline-[3px]
```

Comment:
can we avoid using `[3px]`? Are there any tokens that we can use?

Reply:
actually no, I can extend the theme, so we can use outline class like this: 
'gi-outline-sm' for 3px.

what do you think?

Reply:
yes please, thanks